### DISCUSSION TITLE
What is the reason to use GPL?

### DISCUSSION THREAD

**Discussion by ghost** [Harvey-OS/harvey#1040] [Q&A] [answered]

**Comment by orangecms** [comment]
There are other Plan 9 systems under different licenses. We sought to have one under GPL, which is part of the purpose of the project. Does that answer your question? Maybe we can add that to the website. :)

**Comment by ghost** [comment]
> which is part of the purpose of the project

what is the purpose? Does using an BSD/MIT/ISC license violets the purpose?

**Comment by orangecms** [comment]
Sorry, I think we are heading a bit in a wrong direction. Let's start over. Could you elaborate on your question? I.e., do you seek to use Harvey for something that would violate the license?

**Comment by sevki** [comment, marked as answer]
Harvey is a fork of the GPL release of [Plan 9](https://github.com/brho/plan9) thus it has to inherit that license. 

So yes, re-licensing Harvey with BSD/MIT/ISC would be a violation of the original GPL release, as Bell Labs holds the copyright of our derived work.

Plan 9 source code for a very long while was only distributed under the Lucent Public License and if I recall correctly a lot of people worked hard and called in favours to get a GPL'd version released (i believe for akaros), perhaps we can get @rminnich to tell that story sometime.  So for all intents and purposes the license is set in stone and **CANNOT** change.

**Reply by ghost** [reply]
> Plan 9 source code for a very long while was only distributed under the Lucent Public License and if I recall correctly a lot of people worked hard and called in favours to get a GPL'd version released

What was wrong with LPL https://9p.io/plan9/license.html ?
